project/REVIEWING.md

```diff
@@ -0,0 +1,170 @@
+
+Possible transitions from this state:
+
+ * Close
+ * `1-needs-design-review`: raises design concerns
+ * `2-needs-code-review`: requires more code changes
+ * `4-ready-to-merge`: general case
+
+## Merge - `4-needs-merge`
+
+Maintainers are expected to merge this pull request as soon as possible.  They can ask for a
+rebase, or carry the pull request themselves.  These should be the easy PRs to merge.
+
+Possible transitions from this state:
+
+ * Merge: general case
+ * Close: carry PR
+
+When merging a pull request, the maintainer should consider applying one or multiple impact labels
+to ease future classification (see below).
+
+# Escalation process
+
+Despite the previously described reviewing process, some PR might not show any progress for various
+reasons:
+
+ - No strong opinion for or against the proposed patch
+ - Debates about the proper way to solve the problem at hand
+ - Lack of consensus
+ - ...
+
+All these will eventually lead to stalling PR, where no apparent progress is made accross several
+weeks, or even months.
+
+Maintainers should use their best judgement and apply the `status/needs-attention` label. It must
+be used sparingly, as each PR with such label will be discussed by a group of maintainers during a
+review session. The goal of that session is to agree on one of the following outcomes for the PR:
+
+ * Close, explaining the rationale for not pursuing further
+ * Continue, either by pushing the PR further in the workflow, or by deciding to carry the patch
+   (ideally, a maintainer should be immediately assigned to make sure that the PR keeps continued
+   attention)
+ * Escalate to Solomon by formulating a few specific questions on which his answers will allow
+   maintainers to decide.
+
+# Labels
+
+Labels are carefully picked to optimize for:
+
+ - Readability: maintainers must immediately know the state of a PR
+ - Filtering simplicity: different labels represent many different aspects of
+   the reviewing work, and can even be targetted at different maintainers groups.
+
+A pull request should only be attributed labels documented in this section: other labels that may
+exist on the repository should apply to issues.
+
+## DCO labels
+
+ * `dco/no`: automatically set by a bot when one of the commits lacks proper signature
+
+## Status labels
+
+ * `0-needs-triage`
+ * `1-needs-design-review`
+ * `2-needs-code-review`
+ * `3-needs-docs-review`
+ * `4-ready-to-merge`
+
+Special status labels:
+
+ * `status/needs-attention`: calls for a collective discussion during a review session
+
+## Specialty group labels
+
+Those labels are used to raise awareness of a particular specialty group, either because we need
+help in reviewing the PR, or because of the potential impact of the PR on their work:
+
+ * `group/distribution`
+ * `group/networking`
+ * `group/security`
+ * `group/windows`
+
+## Impact labels
+
+When merging a PR, the maintainer should use his best judgment in applying some labels to help with
```

Comment:
Same here s/merging/reviewing/ ? I think these can be applied at any stage of the review process